ui/packages/lib/src/components/nav_drawer/NavDrawer.js

```diff
@@ -52,29 +39,123 @@ export const NavDrawer = ({ homeUrl = "/", appLinks }) => {
     }
   ];
 
+  const docLinks = [
```

Comment:
This has to be configurable and I don't think we should include link to gojek internal documentations

Reply:
+1 on making it configurable.

Reply:
🤦 just realised this. Should we make it configurable from db?